### PR TITLE
resource location alignment to resource group location

### PR DESCRIPTION
**Pull request by v-fearam**

### CODE REVIEW

**Comment by ckittel** [comment]
Also, we do have work items (tasks) on the backlog for this work (on this repo and the other that you did), so if you complete them, please make sure you close the related tasks.  Guessing you know this already, but mentioning it _just in case_.

File: README.md [hunk 1]
```diff
@@ -32,22 +32,22 @@ az login
 az deployment sub create --name workload-stamp-prereqs --location eastus --template-file ./workload-stamp-prereqs.bicep
```
[the remark this answers is not in the document]
Comment (v-fearam) [reply README.md]: done

**Comment by v-fearam** [comment]
> Also, we do have work items (tasks) on the backlog for this work (on this repo and the other that you did), so if you complete them, please make sure you close the related tasks. Guessing you know this already, but mentioning it _just in case_.

Yes, I did the change only in task assigned to me.

File: README.md [hunk 2]
```diff
@@ -20,6 +20,14 @@ The Drone Delivery application is a sample application that consists of several
 
 ## Deploy an Azure Container Registry (ACR)
 
+## Azure Resources Provisioning
+
```
Comment (ckittel) [inline README.md:24]:
```suggestion
```

**Review by ckittel — APPROVED** [review]

**Pull request merged**